wordpress-mobile/WordPress-iOS
# Add rule to check Podfile against commit hashes

This PR adds a Danger rule that checks the Podfile and verifies that no dependency is referenced by a pointer to a commit hash.

I'm keeping this very simple and the rule just checks for the _commit_ keyword outside of a comment line. I can't think about any issue with this, but if you see any case that could raise a false-positive, please let me know.

To test:
1. commit a Podfile with one or more dependencies referenced by a hash;
2. wait for the CI to complete and check that the warning is raised;
3. revert the commit made in point 1.

Jan, can I bother you with this review?
